python/pyspark/sql/dataframe.py

```diff
@@ -1746,7 +1746,7 @@ def toPandas(self):
             pdf = pd.DataFrame.from_records(self.collect(), columns=self.columns)
 
             for f, t in dtype.items():
-                pdf[f] = pdf[f].astype(t, copy=False)
```

Comment:
Btw, seems to me we call `astype`  on `Series` instead of `DataFrame`? `Series.astype` has `copy` parameter until 0.13.0.

http://pandas.pydata.org/pandas-docs/version/0.13.0/generated/pandas.Series.astype.html

http://pandas.pydata.org/pandas-docs/version/0.12.0/generated/pandas.Series.astype.html